core-bundle/src/Resources/contao/library/Contao/Database/Statement.php

```diff
@@ -141,13 +140,6 @@ public function prepare($strQuery)
 	 */
 	public function set($arrParams)
 	{
-		if (substr_count($this->strQuery, '%s') !== 1 || !\in_array(strtoupper(substr($this->strQuery, 0, 6)), array('INSERT', 'UPDATE'), true))
-		{
-			trigger_deprecation('contao/core-bundle', '4.13', 'Using "%s()" is only supported for INSERT and UPDATE queries with the "%%s" placeholder. This will throw an exception in Contao 5.0.', __METHOD__);
```

Comment:
Where is the exception thrown?

Reply:
I guess Doctrine will? @ausi ?

Reply:
The `trigger_deprecation()` has to be replaced with a `throw` statement here.